ignite/contrib/metrics/regression/canberra_metric.py

```diff
@@ -36,7 +36,23 @@ class CanberraMetric(_BaseRegression):
 
     .. _`Botchkarev 2018`:
         https://arxiv.org/ftp/arxiv/papers/1809/1809.03006.pdf
-
+    
+    .. testcode::
+
+            def process_function(engine, batch):
```

Comment:
Please have a look to others doctests recently introduced. You can use `default_evaluator` which is defined in https://github.com/pytorch/ignite/blob/master/docs/source/conf.py

It means your code would be simplified 
```python
metric = CanberraMetric()
metric.attach(default_evaluator, 'canberra')
preds = torch.Tensor([[3.8], [9.9], [-5.4], [2.1]])
target = preds * 1.5
state = default_evaluator.run([[preds, target]])
print(state.metrics['canberra'])
```
> 

>

Reply:
Oh I missed that. I have changed to this now.